rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -47,6 +48,7 @@ private BookKeeperServer()
     public static void main(String[] args)
     {
         conf = new Configuration();
+        conf.addResource(new YarnConfiguration());
```

Comment:
Can the respective cluster managers add the resources they need ? In that case, hadoop2 or presto cluster manager can add YarnConfiguration as resource and it is going to work for hadoop1 as well